Make.config

```diff
@@ -46,9 +46,9 @@ IOS_PACKAGE_VERSION_BUILD=$(IOS_COMMIT_DISTANCE)
 IOS_PACKAGE_UPDATE_ID=$(shell printf "2%02d%02d%02d%03d" $(IOS_PACKAGE_VERSION_MAJOR) $(IOS_PACKAGE_VERSION_MINOR) $(IOS_PACKAGE_VERSION_REV) $(IOS_PACKAGE_VERSION_BUILD))
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=11.0
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode11.app/Contents/Developer
+XCODE_VERSION=11.1
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11.1_GM_Seed.xip
+XCODE_DEVELOPER_ROOT=/Applications/Xcode11.1-GM.app/Contents/Developer
```

Comment:
is that the location on the bots ?
it does not match https://xamarinhq.slack.com/archives/C99ER4A1M/p1569347681015300

Reply:
Yeah I think we went with this https://xamarinhq.slack.com/archives/C03CEMRFL/p1569348227198300

`Xcode111-GM.app`